apns/test/atom_species_and_cell.py

```diff
@@ -24,6 +24,13 @@ def __init__(self, symbol: str, pseudo_dir: str, pptags: list, orbital_dir: str
 Numerical atomic orbital tags: {self.naotags}
 """)
 
+    def connect_ecut_db(self, fname: str):
+        """connect with the ecut database"""
+        import json, os
+        if not os.path.exists(fname): return
+        with open(fname, "r") as f:
+            self.ecut_db = json.load(f)
+        
```

Comment:
no, because it is possible for apns_cache/ecutwfc.json to be non-existent, in this case should print warning or something instead of terminating the workflow